extensions/opentelemetry-exporter-jaeger/deployment/src/main/java/io/quarkus/opentelemetry/exporter/jaeger/JaegerExporterProcessor.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.opentelemetry.exporter.jaeger;
+
+import java.util.function.BooleanSupplier;
+
+import io.quarkus.arc.deployment.AdditionalBeanBuildItem;
+import io.quarkus.deployment.Feature;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.ExecutionTime;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+
+public class JaegerExporterProcessor {
+
+    static class JaegerExporterEnabled implements BooleanSupplier {
+        JaegerExporterConfig.JaegerExporterBuildConfig jaegerExporterConfig;
+
+        public boolean getAsBoolean() {
+            return jaegerExporterConfig.enabled;
+        }
+    }
+
+    @BuildStep(onlyIf = JaegerExporterEnabled.class)
+    FeatureBuildItem feature() {
+        return new FeatureBuildItem(Feature.OPENTELEMETRY_JAEGER_EXPORTER);
+    }
```

Comment:
Not everything seems to take that approach, and it seems a bit odd to say a Feature is present when it's disabled.

Doesn't that cause issues with anyone thinking it should be working because it's listed, even if it's disabled?

Reply:
> Not everything seems to take that approach

All other features that I know take this approach : opentracing, mertrics, oauth2, ... 
For me, if I add an extension, it provies a feature, wheter it's enabled or not, the feature is provided.

Reply:
micrometer doesn't work that way

Reply:
Neither does smallrye metrics. We only emit the feature if metrics is enabled.. otherwise it is like the feature doesn't exist.

Reply:
OK, maybe we need to align all extensions to do the same (wether its always add the feature build item or not) to have a common experience.
I'll open a discussion on the mailing list later if I don't forget ;)